op-chain-ops/genesis/config.go

```diff
@@ -830,6 +833,9 @@ type DeployConfig struct {
 
 	// DeployCeloContracts indicates whether to deploy Celo contracts.
 	DeployCeloContracts bool `json:"deployCeloContracts"`
+
+	// InitializeAnchorStateRegistry indicates whether to initialize the AnchorStateRegistry.
+	InitializeAnchorStateRegistry bool `json:"initializeAnchorStateRegistry"`
```

Comment:
Where is this coming from?

Reply:
As in, in which upstream commit was that added?

And, why do we have those fields in the config at all? I'd expect them not to be here if they aren't handled properly.

Reply:
@jcortejoso I removed your commits for now, I'll push them in a different branch.